src/main/webapp/admin_dashboard_script.js

```diff
@@ -1,30 +1,158 @@
-google.charts.load('current', {'packages':['line']});
-google.charts.setOnLoadCallback(drawChart);
+var map;
+var styledMapType;
+var map_key = keys.MAP_KEY;
+
+load(`https://maps.googleapis.com/maps/api/js?key=${map_key}`); // Add maps API to html
+
+google.charts.load("current", { packages: ["line"] });
 
+google.charts.setOnLoadCallback(drawChart);
 window.addEventListener("resize", drawChart);
+window.addEventListener("load", drawMap);
+
+//window.onload(drawMap());
```

Comment:
I think the commented line is no longer needed, you could remove this line. Just make `drawMap()` function an onload function for the `admin-dashboard.html` file.